low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -46,7 +46,7 @@ SSPL_LL_SETTING:
 
 INGRESSPROCESSOR:
    consumer_id: sspl_actuator
-   consumer_group: cortx_monitor
+   consumer_group_prefix: cortx_monitor_
```

Comment:
_ added to append node-id ? append as "-node-id" only then, keeping it clean here "cortx_monitor"

Reply:
Done